.golangci.yml

```diff
@@ -59,10 +102,24 @@ linters:
     - whitespace
 
 issues:
-    # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
-    max-same-issues: 0
-    # Excluding configuration per-path, per-linter, per-text and per-source
-    exclude-rules:
-      - path: edge
-        linters:
-          - errcheck
+  # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
+  max-same-issues: 0
+  # Excluding configuration per-path, per-linter, per-text and per-source
+  exclude-rules:
```

Comment:
I'm not very clearly know why we need exclude edge-errcheck rules before?  But I guess with new edged (from v1.12), we may no longer need to set this rules? 
Would you mind try to remove this setting and re-run golint test in this PR?

Reply:
Lint checked failure, so many errors are not checked. Shoud we handle these errs or recover this rule?

Reply:
I think we should handle these errors. If there are a large num of errs, we can first open an issue and split this task by directory, and then solve it together.

Reply:
I'll try to handle it first.